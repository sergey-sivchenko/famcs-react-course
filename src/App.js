import logo from './logo.svg';
import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Here is the very beginning of our TODOs application.
        </p>
      </header>
    </div>
  );
}

export default App;
