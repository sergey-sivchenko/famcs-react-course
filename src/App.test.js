import { render } from "@testing-library/react";

import App from "./App";

test("renders correctly", () => {
  const { container } = render(<App />);
  expect(container.firstChild).toMatchSnapshot();
});
