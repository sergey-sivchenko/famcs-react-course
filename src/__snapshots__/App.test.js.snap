// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-content: center;
  -ms-flex-line-pack: center;
  align-content: center;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  height: 50px;
  width: 300px;
}

.c1 {
  width: 24px;
  height: 24px;
  margin-right: 20px;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

.c2 {
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

<div
  class="App"
>
  <div
    class="todo-container"
  >
    <div
      class="c0"
    >
      <img
        alt="checkmark"
        class="c1"
        src="check-mark.png"
      />
      <div
        class="c2"
      >
        Create static page
      </div>
    </div>
    <div
      class="c0"
    >
      <img
        alt="checkmark"
        class="c1"
        src="close.png"
      />
      <div
        class="c2"
      >
        Create dynamic page
      </div>
    </div>
    <div
      class="c0"
    >
      <img
        alt="checkmark"
        class="c1"
        src="close.png"
      />
      <div
        class="c2"
      >
        Add redux
      </div>
    </div>
  </div>
</div>
`;
